README.md

```diff
@@ -53,6 +53,96 @@ We will pull changes from upstream, so if you want to make a plugin for Vencord
    - Run `automaticInstallDist.bat` and type `a`.
    - After the build process is complete, the software should be ready to use.
 
+## Installation instructions (unofficially and primarily for Linux)
+### Compiling source
+[BetterVencord](https://github.com/Davilarek/Vencord) needs to be compiled. The following below are build instructions for [Linux](https://github.com/Vendicated/Vencord/blob/main/.github/workflows/build.yml):
+```
+git clone https://github.com/Davilarek/Vencord.git
+cd Vencord
+pnpm install --frozen-lockfile
+pnpm buildWeb
```

Comment:
By the way, this is only useful if you intent to use BV on web, like ArmCord, or Discord in a browser